lib/malloy/server/controller.hpp

```diff
@@ -50,6 +52,11 @@ namespace malloy::server
              * to the working directory.
              */
             std::filesystem::path doc_root = ".";
+
+            /**
+             * @brief Agent string used for websocket connections
```

Comment:
This field value is also used for HTTP connections - not only WS connections, right?